lib/btree.js

```diff
@@ -43,7 +43,9 @@ const binarySearch = (node, value) => {
 };
 
 function* inorderTraversal(start, finish, currNode) {
-  let [found, startIndex] = binarySearch(currNode, start);
+  const binarySearchResult = binarySearch(currNode, start);
+  const found = binarySearchResult[0];
```

Comment:
There is no need to have this as a variable, you only use it once. Just inline it.